test/ssl_old_test.c

```diff
@@ -1296,7 +1296,7 @@ int main(int argc, char *argv[])
     }
 
     if (print_time) {
-        if (bio_type != BIO_PAIR) {
+        if (bio_type == BIO_MEM) {
```

Comment:
Why this change?

Reply:
I want to measure the time using -ipv4/6 -time
and this overrides the bio_type == BIO_IPV4, BIO_IPV6
which are clones of BIO_PAIR and are able to do the timimgs,
in contrast to BIO_MEM, which is not doing any timimgs.

Reply:
But BIO_PAIR is using memory bio, and cannot use KTLS.

Reply:
Ok